static/kepler/kepler-init.js

```diff
@@ -0,0 +1,89 @@
+/* global React, ReactDOM, ReactRedux, Redux, KeplerGl */
+/* eslint no-undef: "error" */
+// based on the example at https://github.com/keplergl/kepler.gl/blob/master/examples/umd-client/index.html
+
+const MAPBOX_TOKEN =
```

Comment:
I'm getting this error when running the kepler experience : 

`
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://events.mapbox.com/events/v2?access_token=pk.eyJ1IjoiYW5kcmVhc2t1bmRpZyIsImEiOiJja3ZxcnlmNXc2ZzUwMnFva2F2a3Q1azU5In0.NrvCU8OKlkwJOVFOgZzTzA. (Reason: CORS request did not succeed)
`

It still seems to work though